configure

```diff
@@ -25,6 +25,24 @@ check_util make
 check_util gcc
 check_util lsblk
 
+SCRIPTPATH=`dirname $0`
+SCRIPTPATH=`realpath $SCRIPTPATH`
+SUBMODULES=(
+    "ocf"
+    "test/functional/test-framework"
```

Comment:
maybe get the list of submodules from git?

Reply:
But you may not have a git tree in your sources.

Reply:
It's true, but then why You print communicate "Do 'git submodule update --init' and try again!" below? If somebody will download zip or tarball, then he/ she would not be able to use 'git' commands from here before initiating this directory as git repo, but in this case, he/she would have submodules, so this check wouldn't result with false. This case is probable in case when somebody download OCL from GitHub with git, so I think Adam is right. You should take this list from file '.gitmodules'.

Reply:
I am convinced.